apps/vaccitron/src/app/report-list-page/report-list-page.component.ts

```diff
@@ -1,13 +1,24 @@
-import { Component, Input, OnDestroy, OnInit } from '@angular/core';
+import { Component, Inject, Input, OnDestroy, OnInit } from '@angular/core';
```

Comment:
Report list data should be refreshed immediately after login, so the user gets the fresh data and do not need to wait.